pkg/schedule/schedulers/metrics.go

```diff
@@ -156,15 +156,15 @@ var (
 			Subsystem: "scheduler",
 			Name:      "store_slow_trend_action_status",
 			Help:      "Store trend scheduler calculating actions",
-		}, []string{"reason"})
+		}, []string{"type", "status"})
```

Comment:
should the grafana.json  update else?

Reply:
Nope, this part should be polished in next pr(we need another pr to tidy all related metrics, to make it more clear to read)